packages/utils/odsp-doclib-utils/src/odspDocLibUtils.ts

```diff
@@ -13,11 +13,17 @@ export function isSharepointURL(server: string) {
 }
 
 export function getSharepointTenant(server: string) {
-    let tenant = server.substr(0, server.indexOf("."));
-    if (tenant.endsWith("-my")) {
-        tenant = tenant.substr(0, tenant.length - 3);
+    let tenantName = server.substr(0, server.indexOf(".")).toLowerCase();
+    if (tenantName.endsWith("-my")) {
+        tenantName = tenantName.substr(0, tenantName.length - 3);
     }
-    return tenant === "microsoft" ? "organizations" : `${tenant}.onmicrosoft.com`;
+
+    let restOfTenatHostname = server.substr(tenantName.length).toLowerCase();
```

Comment:
nit: spelling of tenant
```suggestion
    let restOfTenantHostname = server.substr(tenantName.length).toLowerCase();
```

Reply:
Thanks for catching, I fixed it as part of another refactoring that I published in 3rd iteration of this PR